### DISCUSSION TITLE
Plot not showing up in Jupyter notebook (VSCode)

### DISCUSSION THREAD

**Discussion by krunolp** [tpvasconcelos/ridgeplot#213] [Q&A] [answered]
Hi,

When I run the code from the example in my Jupyter Notebook using VSCode, I do not see any output:

<img width="1165" alt="image" src="https://github.com/tpvasconcelos/ridgeplot/assets/68286264/73661c19-8ced-457e-a961-fec187907e91">

Do you maybe know what might be causing this?

**Comment by tpvasconcelos** [comment, marked as answer]
Hi @krunolp! I think this might be a problem with VSCode and not `ridgeplot`.

Can you try running this simple plotly example and let me know if it works for you?

```python
import plotly.express as px

fig = px.line(x=["a","b","c"], y=[1,3,2], title="sample figure")
fig.show()
```

You should see this:

<img width="400" alt="image" src="https://github.com/tpvasconcelos/ridgeplot/assets/17701527/9e56fc03-ab00-40fa-8472-98d89efd41c4">

If you don't maybe, this could help you: https://stackoverflow.com/questions/64849484/display-plotly-plot-inside-vs-code

As a workaround, you could try to call `fig.show("browser")` to open the figure in a new browser tab.

**Comment by krunolp** [comment]
Thank you @tpvasconcelos for the prompt answer, both of your answers solved my problem! The workaround is particularly neat!
